package.json

```diff
@@ -16,34 +16,48 @@
     "cross",
     "platform"
   ],
+  "engines": {
+    "node": ">=18 <20"
+  },
+  "files": [
+    "index.js",
+    "src",
+    "docs",
+    "example",
+    "prebuilds",
+    "binding.gyp",
+    "LICENSE",
+    "README.md",
+    "requirements.txt"
+  ],
   "directories": {
     "test": "tests"
   },
   "scripts": {
     "configure": "node-gyp configure",
-    "build": "node-gyp build",
+    "build": "node-gyp-build && npm run prebuildify",
+    "prebuildify": "prebuildify -t 18.18.2 -t 20.5.1 --strip",
     "rebuild": "node-gyp rebuild",
     "readme": "jsdoc2md --template docs/README.hbs index.js > README.md",
     "lint-js": "eslint *.js",
     "lint-cpp": "cpplint --recursive src",
     "lint": "npm run lint-js && npm run lint-cpp",
     "test": "mocha tests -R spec",
-    "install": "prebuild-install || node-gyp rebuild",
-    "prebuild-release": "prebuild --all --strip"
+    "install": "node-gyp-build"
```

Comment:
Should this be a `prepare` script?

Reply:
I'm not sure about that one. I took it straight out of the prebuildify documentation. What would be the benefit of a using a prepare script instead?